<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1322</width>
    <height>773</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>.AppleSystemUIFont</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>71</y>
         <width>396</width>
         <height>46</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="StartOfTrackButton">
          <property name="toolTip">
           <string>Start of Track</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="MaterialSymbolsIcons.qrc">
            <normaloff>:/resources/MaterialSymbols/skip_prev_24dp_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/resources/MaterialSymbols/skip_prev_24dp_FILL0_wght400_GRAD0_opsz24.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BackwardsButton">
          <property name="toolTip">
           <string>Seek Backwards</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="MaterialSymbolsIcons.qrc">
            <normaloff>:/resources/MaterialSymbols/fast_rewind_24dp_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/resources/MaterialSymbols/fast_rewind_24dp_FILL0_wght400_GRAD0_opsz24.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PausePlay">
          <property name="toolTip">
           <string>Play</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="MaterialSymbolsIcons.qrc">
            <normaloff>:/resources/MaterialSymbols/play_circle_24dp_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/resources/MaterialSymbols/play_circle_24dp_FILL0_wght400_GRAD0_opsz24.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StopButton">
          <property name="toolTip">
           <string>Stop</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="MaterialSymbolsIcons.qrc">
            <normaloff>:/resources/MaterialSymbols/stop_circle_24dp_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/resources/MaterialSymbols/stop_circle_24dp_FILL0_wght400_GRAD0_opsz24.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ForwardButton">
          <property name="toolTip">
           <string>Seek Forward</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="MaterialSymbolsIcons.qrc">
            <normaloff>:/resources/MaterialSymbols/fast_forward_24dp_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/resources/MaterialSymbols/fast_forward_24dp_FILL0_wght400_GRAD0_opsz24.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="EndOfTrackButton">
          <property name="toolTip">
           <string>End of Track</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="MaterialSymbolsIcons.qrc">
            <normaloff>:/resources/MaterialSymbols/skip_next_24dp_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/resources/MaterialSymbols/skip_next_24dp_FILL0_wght400_GRAD0_opsz24.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>195</y>
         <width>611</width>
         <height>42</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="CustomButton" name="CueButton_1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;
background-color: rgb(128, 128, 128);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomButton" name="CueButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;
background-color: rgb(128, 128, 128);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomButton" name="CueButton_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;
background-color: rgb(128, 128, 128);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomButton" name="CueButton_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;
background-color: rgb(128, 128, 128);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomButton" name="CueButton_5">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;
background-color: rgb(128, 128, 128);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomButton" name="CueButton_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;
background-color: rgb(128, 128, 128);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomButton" name="CueButton_7">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;
background-color: rgb(128, 128, 128);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CustomButton" name="CueButton_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>20</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;
background-color: rgb(128, 128, 128);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>155</y>
         <width>611</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>Shift</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="HotCueRadioButton">
          <property name="text">
           <string>Hot Cue</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="LoopRadioButton">
          <property name="text">
           <string>Loop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RollRadioButton">
          <property name="text">
           <string>Roll</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="SlicerRadioButton">
          <property name="text">
           <string>Slicer</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>401</width>
         <height>42</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QScrollBar" name="horizontalScrollBar">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="Deck" name="Deck1" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="Deck" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1322</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDJController">
    <property name="title">
     <string>DJController</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Recent"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuDJController"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_Recent">
   <property name="text">
    <string>Open Recent</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit DJController</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomButton</class>
   <extends>QPushButton</extends>
   <header>custombutton.h</header>
  </customwidget>
  <customwidget>
   <class>Deck</class>
   <extends>QWidget</extends>
   <header location="global">deck.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MaterialSymbolsIcons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
